naming/src/main/java/com/alibaba/nacos/naming/healthcheck/extend/HealthCheckExtendProvider.java

```diff
@@ -0,0 +1,92 @@
+package com.alibaba.nacos.naming.healthcheck.extend;
+
+import com.alibaba.nacos.api.naming.pojo.AbstractHealthChecker;
+/*
```

Comment:
licence info is alway on the top of the file.

Reply:
OK